packages/devtools_app/lib/src/screens/logging/logging_screen_v2/logging_table_v2.dart

```diff
@@ -66,16 +58,15 @@ class _LoggingTableV2State extends State<LoggingTableV2> {
           children: [
             Expanded(
               child: DevToolsClearableTextField(
-                labelText: 'Filter',
-                onSubmitted: (value) {},
+                labelText: 'Search',
               ),
```

Comment:
Can you use `SearchField` here instead? Fine to do as part of a separate PR, but can you add a TODO?